opm/simulators/wells/StandardWellEquations.cpp

```diff
@@ -376,13 +376,13 @@ extractCPRPressureMatrix(PressureMatrix& jacobian,
             bweights[0][blockSz-1] = 1.0;
             diagElem = 1.0; // better scaling could have used the calculation below if weights were calculated
         } else {
-            for (std::size_t i = 0; i < cell_weights.size(); ++i) {
+            for (std::size_t i = 0; i < blockSz; ++i) {
                 bweights[0][i] = cell_weights[i];
```

Comment:
this broke basically every test when running with debug iterators. cell_weights is of length 3, blockSz is 4.

Reply:
it's a bit involved. fundamentally it's just adding -D_GLIBCXX_DEBUG to CXX flags. however, you may or may not have to build dependencies using the same flag. try without, if you don't get linker errors, then all is fine. I will email you some more detailed instructions for rainier days.

Reply:
I have a proposed fix, will push PR.